riscv64-purecap.cmake

```diff
@@ -4,20 +4,20 @@ set(CMAKE_SYSTEM_NAME Linux)
 set(SDK "$ENV{HOME}/cheri/output/sdk" CACHE PATH "path to cheri SDK")
 
 # Set toolchain compilers
-set(CMAKE_C_COMPILER ${SDK}/bin/riscv64-unknown-freebsd13-cc)
-set(CMAKE_CXX_COMPILER ${SDK}/bin/riscv64-unknown-freebsd13-c++)
+set(CMAKE_C_COMPILER ${SDK}/bin/clang)
+set(CMAKE_CXX_COMPILER ${SDK}/bin/clang++)
 
 # Don't run the linker on compiler check
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
 
 # Define sysroot path for CHERI-build`
-set(CMAKE_SYSROOT ${SDK}/sysroot-riscv64-purecap)
+# set(CMAKE_SYSROOT ${SDK}/sysroot-riscv64-purecap)
```

Comment:
Why is this commented out ?

Reply:
Oh you don't need to set the sysroot manually anymore, it is integrated in the cfg file now.